format.md

```diff
@@ -124,6 +133,29 @@ is a list of xy vertices. The child name of the outer list should be "polygons";
 the child name of the middle list should be "rings"; the child name of the
 inner list should be "vertices".
 
+### Serialized encodings
+
+**Well-known binary (WKB)**: `Binary` or `LargeBinary` or `FixedSizeBinary`
+
+It may be useful for implementations that already have facilities to read
+and/or write well-known binary (WKB) to store features in this form without
+modification. When well-known binary is stored in an Arrow array, it should
+follow the convensions defined in the
+[GeoParquet specification](https://github.com/opengeospatial/geoparquet).
+Notably, it should use the ISO form instead of EWKB when Z or M dimensions
+are included and axis order is defined as easting, northing/longitude, latitude
+regardless of the order specified by the coordinate system. Producers may store
+EWKB or invalid WKB in a GeoArrow WKB array and consumers may error if they
```

Comment:
I suppose this is meant to support Postgis? 
While in practice many WKB readers won't care which format it gets (such as GEOS), it does sound a bit strange to first explicitly say that it should be ISO format and not EWKB, to then say it is allowed to also store EWKB. 

You raised the question in the top post as well, but should we have a separate "ewkb" type? (that would allow the spec to be stricter, but maybe in practice not that useful)

Reply:
IMO I'd rather have that information on the extension type than have to infer it. So I'd much rather have `geoarrow.wkb` and `geoarrow.ewkb` (there's also already `ogc.wkb` that GDAL has produced since 3.6 in its arrow API)

Reply:
Adding it as metadata is maybe indeed a good idea. Then for those applications that don't need to distinguish between both only have a single type to deal with, while if you do care about it, you have a place to store information about it.

One additional question for EWKB is what to do with the CRS in the metadata vs the SRID that can be embedded in the EWKB value. Do we want to say something that if CRS metadata is defined, this gets precedence, or do we leave this fully application dependent?